### PR TITLE
[benchmark] Fix the swiftpm based benchmark build.

### PR DESCRIPTION
**Pull request by gottesmm**
This does a few things:

1. We were not updated for libProc's addition. I bumped the swiftpm version
number to get the systemLibrary functionality (thanks Ankit).

2. I split up a bunch of lines to help the typechecker out a little bit.

### CODE REVIEW

**Comment by gottesmm** [comment]
This is NFC since this is only used for offline development in Xcode right now (hopefully we just switch to using this in general).

**Comment by gottesmm** [comment]
@swift-ci smoke test

**Review by aciidgh — APPROVED** [review]
This can be further simplified but as you said one step at a time 😀

File: benchmark/Package.swift [hunk 1]
```diff
@@ -39,50 +39,57 @@ let multiSourceLibraries: [String] = {
...
-            ] + singleSourceLibraries.map { .target(name: $0) }
-              + multiSourceLibraries.map { .target(name: $0) },
-            path: "utils",
-            sources: ["main.swift"]),
-    .target(name: "ObjectiveCTests",
-      path: "utils/ObjectiveCTests",
-      publicHeadersPath: "."),
-  ] + singleSourceLibraries.map { x in
+    dependencies: [
+      .target(name: "TestsUtils"),
```
Comment (aciidb0mb3r) [inline benchmark/Package.swift]: You can use the shorthand and just use "TestsUtils" string directly. SwiftPM can figure out that you're referring to a target.

File: benchmark/Package.swift [hunk 2]
```diff
@@ -39,50 +39,57 @@ let multiSourceLibraries: [String] = {
...
-      sources: ["TestsUtils.swift"]),
-    .target(name: "DriverUtils",
-      dependencies: [.target(name: "TestsUtils")],
-      path: "utils",
-      sources: ["DriverUtils.swift", "ArgParse.swift"]),
+var products: [Product] = []
+products.append(.library(name: "TestsUtils", type: .static, targets: ["TestsUtils"]))
+products.append(.library(name: "DriverUtils", type: .static, targets: ["DriverUtils"]))
+products.append(.library(name: "ObjectiveCTests", type: .static, targets: ["ObjectiveCTests"]))
+products.append(.executable(name: "SwiftBench", targets: ["SwiftBench"]))
```
Comment (aciidb0mb3r) [inline benchmark/Package.swift]: You only need this product, rest can be discarded.

**Pull request merged**